src/main/java/mightyduck/controller/MainController.java

```diff
@@ -0,0 +1,115 @@
+package mightyduck.controller;
+
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import mightyduck.MightyDuck;
+import mightyduck.command.CommandResult;
+import mightyduck.command.CommandResultType;
+import mightyduck.data.task.Task;
+import mightyduck.exception.InvalidStoragePathException;
+import mightyduck.exception.StorageLoadException;
+import mightyduck.utils.Pair;
```

Comment:
Maybe can try adding a blank line between the javafx and mightyduck import statements to improve readability?
```suggestion
import javafx.application.Platform;
import javafx.fxml.FXML;
import javafx.scene.control.Alert;
import javafx.scene.control.Alert.AlertType;
import javafx.scene.control.Button;
import javafx.scene.control.ScrollPane;
import javafx.scene.control.TextField;
import javafx.scene.image.Image;
import javafx.scene.layout.AnchorPane;
import javafx.scene.layout.VBox;

import mightyduck.MightyDuck;
import mightyduck.command.CommandResult;
import mightyduck.command.CommandResultType;
import mightyduck.data.task.Task;
import mightyduck.exception.InvalidStoragePathException;
import mightyduck.exception.StorageLoadException;
import mightyduck.utils.Pair;
```

Reply:
I followed the tutorial on how to configure the code style on IntelliJ.

![image](https://github.com/user-attachments/assets/9884c262-f59a-4926-9540-66e676a1dbe8)

Based on the image, I believe there is no harm in omitting the blank line. But thanks regardless!